TMinus/Launch.swift

```diff
@@ -29,6 +30,31 @@ struct Launch {
     //MARK: Private
 }
 
+extension Launch: NotificationType {
+    
+    func makeNotificationRequest() -> UNNotificationRequest {
+        let date = startDate.addingTimeInterval(-.oneHour)
+        let components = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second], from: date)
+        
+        let content = UNMutableNotificationContent()
+        var title = rocketName
```

Comment:
A small improvement can be made to avoid the mutable state for 'title'.

```
let title = missionName.flatMap {
    $0.isEmpty ? nil : "\($0) - \(missionName)"
} ?? rocketName

content.title = title
```